src/components/UI/organisms/AddMealForm/AddMealForm.css

```diff
@@ -0,0 +1,8 @@
+.AddMealForm {
```

Comment:
I saw that Filip recommend you using CSS modules. If you decide to use them, please make also this file a CSS module.